react-toggle-switch-component/src/toggleSwitch.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+class ToggleSwitch extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {clicked: false, classColor: 'default', onOff: 'On'}
+        this.toggle = this.toggle.bind(this);
+    }
+
+    toggle(){
+        if(this.state.clicked === true){
```

Comment:
Instead of checking `clicked` here you can just check `this.state.onOff === 'On'`. And since you aren't using `clicked` anywhere else you can remove `clicked` from state.